client/components/ScenarioEditor/index.jsx

```diff
@@ -14,8 +14,6 @@ class ScenarioEditor extends Component {
             categories: []
         };
 
-        this.fetchScenario = this.fetchScenario.bind(this);
```

Comment:
I ended up finding that this `fetchScenario` is redundant with the one in the top level editor and was causing me some state issues when copying.